docs/cvl/cvl2/changes.md

```diff
@@ -857,3 +857,91 @@ was not used.  It has been removed in CVL 2.
 % If you do not change this, you will see the following error:
 % ```
 
+Changes to the Command Line Interface (CLI)
+-------------------------------------------
+
+As part of the transition to CVL 2.0 some changes were made to the Command-Line Interface (CLI) for enhanced clarity, 
+uniformity, and readability.
+
+### Flags Renaming
+
+In CVL 2.0 some flags were renamed. Flags that were renamed are:
+1. flags with names that are too generic or wrong
+2. flags that do not match their corresponding key in the conf file
+3. flags that do not follow the snake case format
+
+This is the list of the flags that were renamed:
+
+| CVL 1.0        | CVL 2.0              |
+|----------------|----------------------|
+| --settings     | --prover_args        |
+| --path         | --solc_allow_path    |
+| --optimize     | --solc_optimize      |
+| --optimize_map | --solc_optimize_map  |
+| --get_conf     | --conf_output_file   |
+| --assert       | --assert_contracts   |
+| --bytecode     | --bytecode           |
+| --coinbaseMode | --coinbase_mode      |
```

Comment:
I don't think we want this flag mentioned in our docs. More generally, no hidden flag should be mentioned here

Reply:
removed.